docs/source/index.html.erb

```diff
@@ -1,3 +1,5 @@
+<% @hide_github_badge = true %>
```

Comment:
(minor) Try not to make readers think in double negatives (e.g. `@hide_github_badge = false`). Easier to grok if it was `show_github_badge = true`).